docs/Advanced/SQL Searches/SQL WHERE Clause.md

```diff
@@ -0,0 +1,44 @@
+---
+publish: true
+---
+
+# WHERE Clause
+
+<span class="related-pages">#advanced/sql-search</span>
+
+## Operators in The WHERE Clause
+
+The following basic operators can be used in the `WHERE` clause. More can be found on the [[SQL Operators]] page.
+
+| Operator | Description                                                                 |
+| -------- | --------------------------------------------------------------------------- |
+| =        | Equal                                                                       |
+| >        | Greater than                                                                |
+| <        | Less than                                                                   |
+| >=       | Greater than or equal                                                       |
+| <=       | Less than or equal                                                          |
+| <>       | Not equal. Note: In some versions of SQL this operator may be written as != |
```

Comment:
Probably all that matters here is what the SQL version used by Tasks does...

Reply:
updated.